### DISCUSSION TITLE
Viewing scores after playing beatmap causes unhandled realm error?

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16620] [Q&A] [answered]
Definitely not qualified to start dissecting what's happening here...
Version local release; latest commit cdd63e428c9ba6797d14433f2c828bb4dbe4ccf8
[network.log.txt](https://github.com/ppy/osu/files/7938991/network.log.txt) | [runtime.log.txt](https://github.com/ppy/osu/files/7938992/runtime.log.txt)

https://user-images.githubusercontent.com/74262407/151105484-94c8a9ce-ebed-4d42-8b03-55fa5c416d5e.mp4

**Comment by peppy** [comment, marked as answer]
Thanks for the report. Tracking via https://github.com/ppy/osu/issues/16621.
